components/common/Label.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from '@chakra-ui/react'
+
+interface Props {
+  color?: string
+  children?: React.ReactNode
+  [x: string]: unknown
```

Comment:
Is this the workaround to allow extending the types for `Box`? Or something else?

Reply:
Its a fairly sane way of allowing any props to pass through as a spread

Reply:
Yup, exactly. To me, this just says, "you can give me other props and Imma forward them on somewhere." I don't know if there's a way to be more specific that "somewhere" is to a `Box` and so they can only include `Box` props? I suppose that would be an improvement.

Reply:
According to some quick diving into the TS for chakra, `BoxProps` is the interface for it. I think to use it what you would have to do is 

```
interface Props extends BoxProps {
  color?: string
  children?: React.ReactNode
}
```
 But i haven't tested that yet, so it might not be 100% correct

Reply:
Nice, thanks! I'll try it out now.

Reply:
Hmmm...technically, `color` and `children` are already props of `Box`, so maybe I can just use `BoxProps` as the type directly? It's interesting to see how coming at this with a JS vs. TS mindset leads to such different places.

Reply:
Ooh if you can just call BoxProps then that should solve it without any extra interface work!